### PR TITLE
define qemu as a qemu2 driver alias

### PR DESCRIPTION
**Pull request by sharifelgamal**
Related: #14146

### CODE REVIEW

**Comment by sharifelgamal** [comment]
cc @afbjorklund

File: pkg/minikube/registry/drvs/qemu2/qemu2.go [hunk 1]
```diff
@@ -118,9 +119,9 @@ func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
 		qemuCPU = "cortex-a72"
 		// highmem=off needed, see https://patchwork.kernel.org/project/qemu-devel/patch/20201126215017.41156-9-agraf@csgraf.de/#23800615 for details
 		if runtime.GOOS == "darwin" {
-			qemuMachine = "virt,highmem=off"
+			qemuMachine += ",highmem=off"
```
Comment (afbjorklund) [inline pkg/minikube/registry/drvs/qemu2/qemu2.go:122]: this change is unrelated

File: pkg/minikube/registry/drvs/qemu2/qemu2.go [hunk 2]
```diff
@@ -118,9 +119,9 @@ func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
...
 		} else if _, err := os.Stat("/dev/kvm"); err == nil {
-			qemuMachine = "virt,gic-version=3"
+			qemuMachine += ",gic-version=3"
```
Comment (afbjorklund) [inline pkg/minikube/registry/drvs/qemu2/qemu2.go:124]: this change is unrelated

**Review by afbjorklund — APPROVED** [review]
Works for me, if wondering the `qemu` driver is for `docker-machine` (similar to the `kvm` driver)

The qemu2 and kvm2 are the internal versions, that are forked and more adapted to minikube...
